frontend/__mocks__/localStorage.ts

```diff
@@ -2,10 +2,10 @@ let _localStorage = {};
 
 (window as any).localStorage = (window as any).sessionStorage = {
   setItem(key, value) {
-    return Object.assign(_localStorage, {[key]: value});
+    Object.assign(_localStorage, { [key]: value} );
   },
   getItem(key) {
-    return _localStorage[key];
+    return _localStorage[key] || null;
```

Comment:
Note: these changes are made so that mock `localStorage` is as close to one used in production as possible.

For example, in case of `getItem` method,

> If the key does not exist, `null` is returned.

https://developer.mozilla.org/en-US/docs/Web/API/Storage/getItem

Reply:
> You should only return null if the key doesn't exist. If the value is an empty string, this will incorrectly return `null`.

Right, I've missed that. Will fix that in a separate PR.

Reply:
> Will fix that in a separate PR.

#1925